src/main/java/injectorpractice/dao/ManufacturerDaoJdbcImpl.java

```diff
@@ -0,0 +1,116 @@
+package injectorpractice.dao;
+
+import injectorpractice.exceptions.DataProcessingException;
+import injectorpractice.lib.Dao;
+import injectorpractice.model.Manufacturer;
+import injectorpractice.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ManufacturerDaoJdbcImpl implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String create = "INSERT INTO manufacturer (manufacturer_name, manufacturer_country)"
```

Comment:
```suggestion
        String create = "INSERT INTO manufacturer (name, country)"
```
You already know it is a manufacturer